{
    "cve": {
        "id": "CVE-2024-0565",
        "sourceIdentifier": "secalert@redhat.com",
        "published": "2024-01-15T20:15:43.630",
        "lastModified": "2024-01-16T13:56:05.467",
        "vulnStatus": "Awaiting Analysis",
        "descriptions": [
            {
                "lang": "en",
                "value": "An out-of-bounds memory read flaw was found in receive_encrypted_standard in fs/smb/client/smb2ops.c in the SMB Client sub-component in the Linux Kernel. This issue occurs due to integer underflow on the memcpy length, leading to a denial of service."
            },
            {
                "lang": "es",
                "value": "Se encontr\u00f3 un fallo de lectura de memoria fuera de los l\u00edmites en receive_encrypted_standard en fs/smb/client/smb2ops.c en el subcomponente SMB Client en el kernel de Linux. Este problema se produce debido a un desbordamiento insuficiente de enteros en la longitud de memcpy, lo que provoca una denegaci\u00f3n de servicio."
            }
        ],
        "metrics": {
            "cvssMetricV31": [
                {
                    "source": "secalert@redhat.com",
                    "type": "Secondary",
                    "cvssData": {
                        "version": "3.1",
                        "vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:R/S:U/C:H/I:H/A:H",
                        "attackVector": "NETWORK",
                        "attackComplexity": "HIGH",
                        "privilegesRequired": "LOW",
                        "userInteraction": "REQUIRED",
                        "scope": "UNCHANGED",
                        "confidentialityImpact": "HIGH",
                        "integrityImpact": "HIGH",
                        "availabilityImpact": "HIGH",
                        "baseScore": 7.1,
                        "baseSeverity": "HIGH"
                    },
                    "exploitabilityScore": 1.2,
                    "impactScore": 5.9
                }
            ]
        },
        "weaknesses": [
            {
                "source": "secalert@redhat.com",
                "type": "Secondary",
                "description": [
                    {
                        "lang": "en",
                        "value": "CWE-191"
                    }
                ]
            }
        ],
        "references": [
            {
                "url": "https://access.redhat.com/security/cve/CVE-2024-0565",
                "source": "secalert@redhat.com"
            },
            {
                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2258518",
                "source": "secalert@redhat.com"
            },
            {
                "url": "https://www.spinics.net/lists/stable-commits/msg328851.html",
                "source": "secalert@redhat.com"
            }
        ]
    }
}